src/cloud/live/sens-data-over.md

```diff
@@ -45,6 +45,8 @@ Because {{site.data.var.ece}} supports only the Magento production and maintenan
 
 Sensitive values are _not_ stored in the `app/etc/config.php` file. Any sensitive configurations export to the `app/etc/config.php` file during the `config:dump` process. We recommend using environment variables to store sensitive values. For an example, see [Add Magento authentication keys][auth].
```

Comment:
sensitive configurations export to the `app/etc/env.php` file, not to `app/etc/config.php`

Reply:
Also, there are some other info which is not clear for me. I cannot add commend for that line, but it for lines 34-36.
```
-  Configured values for settings entered through the Magento Admin store configuration
```
config.php file won't have data which was added through Magento Admin panel until running `dump` command. So, that sentence more false than true.
```
- Extension list
```
I do not understand what is meant under this. List of third-party modules? Then yes, it's true as config.php is the only source about modules states (enabled/disabled)
```
- Scopes value for static content deployment (the default SCD strategy is [quick][])
```
why strategy is mentioned here?

P.S. I apologize that I add comments to not touched in this PR info. Probably we may create new PR for this

Reply:
@NadiyaS - This is outside the remit of this PR. I'd be happy if we can deal with what's in scope for this PR and then open a New PR to overhaul this page - this statement is actually confusing as it's referring to the "ece-tools config:dump" action not the "bin/magento app:config:dump"

If you are okay to look at this page as a whole - I will create an issue right now and we can work on it.

Reply:
@BarnyShergold sure, let's update it in new PR. I will check whole page